rfc/rfc023-x509credential.md

```diff
@@ -0,0 +1,453 @@
+                            └────┬────┘       └────┬─────┘                                        
+                                 │                 │                                              
+                                 │                 │                                              
+┌───────────┐            ┌───────┴───────┐         │                                              
+│  ROOT CA  │            │     UZI       │ ┌───────┴────────┐               ┌────┐                
+└─────┬─────┘            │  Certificate  │ │ X509Credential ┼───────────────► VP │                
+      │                  └───────────────┘ └───────┬──┬─────┘               └─┬──┘                
+      │                          │                 │  │    ┌────────────┐     │     ┌────────────┐
+┌─────┴─────┐  Request   ┌───────┴───────┐         │  │    │            │     │     │            │
+│ Source of ◄────────────┼   Holder of   ┼─────────┴──┼────►   Wallet   ┼─────┴─────►  Verifier  │
+│   Trust   ┌────────────►     Trust     │       Issue│    │            │   Present │            │
+└───────────┘   Issue    └───────────────┘            │    └──────┬─────┘           └────────────┘
+                                                      │           │                               
+                                                      │           │                               
+                                              ┌───────┴─┐    ┌────┴────┐                          
+                                              │ did:web ├────┤ Keypair │                          
+                                              └─────────┘    └─────────┘                          
+```
+
+The main steps in the diagram are:
+
+* The holder generates a keypair and requests a UZI certificate from the UZI register with a Certificate Signing
+  Request (CSR).
+* The UZI register issues the certificate to the holder of the UZI certificate and signs the request with the
+  intermediate CA, linked to the root CA.
+* The holder of the UZI creates a `X509Credential` Verifiable Credential:
+  * The holder set the `did:x509` of the UZI certificate as issuer to the `X509Credential` Verifiable Credential.
+  * The holder includes the complete chain in the `X509Credential` Verifiable Credential.
+  * The holder issues the `X509Credential` Verifiable Credential to its own NUTS identity as `did:web` .
+  * The holder signs the `X509Credential` Verifiable Credential with the keypair associated with the UZI certificate.
+* The holder places the `X509Credential` Verifiable Credential in the wallet.
+* The holder presents the `X509Credential` Verifiable Credential to the verifier, and signs the presentation with the
+  keypair associated with the `did:web` of the holder.
+* The verifier now can verify that:
+  * The `X509Credential` Verifiable Credential is issued by a `did:x509` issued by the the UZI register.
```

Comment:
> a `did:x509` issued by the the UZI register.

This is a bit short through the corner ;)
The did is not issued be the UZI register (CIBG?). It is signed with a private key which is also associated with a certificate issued by the UZI register.